generators/connection/index.js

```diff
@@ -243,6 +244,20 @@ module.exports = class ConnectionGenerator extends Generator {
             return false;
           }
 
+          if(typeof connectionString === 'object' && database === 'rethinkdb') {
```

Comment:
Did this have to be added? It should already be done in https://github.com/feathersjs/generator-feathers/blob/master/generators/connection/index.js#L55